doc/frequent-itemsets.rst

```diff
@@ -0,0 +1,62 @@
+Frequent Itemsets
+=================
+
+.. figure:: icons/frequent-itemsets.png
+
+Finds frequent itemsets in the data.
+
+Signals
+-------
+
+#### Inputs:
+
+-  **Data**
+
+Data set
```

Comment:
We're not changing it _again_. Examine carefully [my first comment](https://github.com/biolab/orange3-associate/pull/1/commits/9da96cd58a9d7fb4712cfcc7d00d8547833937d2#r62174840). Also the resulting HTML.